testing/functional/tests/cypress/integration/student/04-application-page.spec.ts

```diff
@@ -0,0 +1,158 @@
+
+        //On click -> Alert in progress popup open
+        ApplicationObject.startApplicationTuitionWaiverBtn()
+        ApplicationObject.applicationAlreadyInProgressTxt()
+
+        ApplicationObject.startApplicationBCloanForgivenessProgramBtn()
+        ApplicationObject.applicationAlreadyInProgressTxt()
+
+        ApplicationObject.startApplicationPacificLeaderLoanForgivenessBtn()
+        ApplicationObject.applicationAlreadyInProgressTxt()
+
+        ApplicationObject.startApplicationInterestFreeStatusBtn()
+        ApplicationObject.applicationAlreadyInProgressTxt()
+
+    })
+
+    it("By clicking on edit button it redirects to Welcome Page in application form.", () => {
+        ApplicationObject.applicationBtn()
+        ApplicationObject.draftApplication()
+    })
+
+    it("By clicking on Next Section button from Welcome Page it redirects to Program Page in application form.", () => {
+        ApplicationObject.applicationBtn()
+        ApplicationObject.draftApplication()
+        ApplicationObject.NextSectionBtn()
+    })
+
+    it("Check without selecting any mandatory fields in Program section if the user clicks Next Section button then the alert message is displayed or not in application form.", () => {
+        ApplicationObject.applicationBtn()
+        ApplicationObject.draftApplication()
+        ApplicationObject.NextSectionBtn()
+        ApplicationObject.NextSectionBtn()
+        ApplicationObject.errorMsgTxtForSchoolAttending()
+    })
+
+    it("Verify that user must be redirect to previous form by selecting Previous section button in application form.", () => {
+        ApplicationObject.applicationBtn()
+        ApplicationObject.draftApplication()
+        ApplicationObject.NextSectionBtn()
+        ApplicationObject.WelcomeBtn()
+        ApplicationObject.NextSectionBtn()
+        ApplicationObject.PreviousSectionBtn()
+        ApplicationObject.WelcomeTxt()
+    })
+    
+    it("Verify that user able to check the checkbox in Program section in application form.", () => {
+        ApplicationObject.applicationBtn()
+        ApplicationObject.draftApplication()
+        ApplicationObject.NextSectionBtn()
+        ApplicationObject.mySchoolIsNotListedCheckbox()
+        ApplicationObject.checkboxAlertMsg()
+    })
+
+    it("Verify that user able to uncheck the checkbox in Program section in application form.", () => {
+        ApplicationObject.applicationBtn()
+        ApplicationObject.draftApplication()
+        ApplicationObject.NextSectionBtn()
+        ApplicationObject.mySchoolIsNotListedCheckbox()
+        ApplicationObject.uncheckAlertMsg()
+        ApplicationObject.checkboxAlertMsgNotExist()
+    })
+
+    it("Verify that user able to edit all details in Program page in application form.", () => {
+        ApplicationObject.applicationBtn()
+        ApplicationObject.draftApplication()
+        ApplicationObject.NextSectionBtn()
+        ApplicationObject.schoolIWillBeAttendingDropdownValue()
```

Comment:
I personally would prefer avoiding abbreviations for elements in general.
For instance, here I can see the 'Dropdown' used, and in other places, I can see Txt and Btn.
I would prefer to have all of them in an extended way like Dropdown, InputText (or Text), Button, etc.

Reply:
Yes, changes have been made.